core/assets/bundles/bundle_de.properties

```diff
@@ -1108,10 +1115,10 @@ mech.omega-mech.description = Ein klobiger und gut gepanzerter Mech, der für de
 mech.dart-ship.description = Das Standard-Schiff. Einigermaßen schnell und leicht. Hat nur wenig Offensivkraft und geringe Erzabbaugeschwindigkeit.
 mech.javelin-ship.description = Ein Schiff für Überfälle. Anfänglich träge, kann es auf hohe Geschwindigkeiten beschleunigen, um an gegnerischen Außenposten vorbei zu fliegen und dabei mit seinen Blitzwaffen und Raketen große Mengen an Schaden verursachen.
 mech.trident-ship.description = Ein schwerer Bomber, der solide gepanzert ist.
-mech.glaive-ship.description = Ein großes, gut gepanzertes Gunship. Ausgerüstet mit einer Brandwaffe. Gute Beschleunigung und maximale Geschwindigkeit.
-unit.draug.description = Eine primitive Bergbaudrohne. Günstig herzustellen. Entbehrlich. Baut automatisch Kupfer und Blei in der Nähe ab. Bringt abgebaute Ressourcen zum nächstgelegenen Kern.
-unit.spirit.description = Die anfängliche Drohne. Sie baut gewöhnlich in der Basis Erz ab, sammelt Materialien und repariert Blöcke.
-unit.phantom.description = Eine fortgeschrittene Drohne. Baut automatisch Erz ab, sammelt Materialien und repariert Blöcke. Deutlich effizienter als die Standard-Drohne.
+mech.glaive-ship.description = Ein großes, gut gepanzertes Kanonenschiff. Ausgerüstet mit einem Flammenwerfer. Gute Beschleunigung und maximale Geschwindigkeit.
```

Comment:
Kanonenschiff wuerde man wahrscheinlich als tatsaechliches Piratenschiff mit Kanonen interpretieren. In Videospielen ist dies eher Kampfflieger.

Reply:
Piratenschiff kommt mir eher als letztes in Sinn. 
https://de.wikipedia.org/wiki/Kanonenboot
Ich habe einfach Boot mit Schiff ersetzt. Aber je nach Film wird sogar im Weltraum das Wort Boot verwendet.